schemas/schema.appointment.js

```diff
@@ -0,0 +1,53 @@
+const schema = require('mongoose').schema;
+
+const definition = {
+	"_appointmentId": String,
+	"dentistId": {
+		"type": String,
+		"required": String,
+	},
+	"patientId": {
+		"type": String,
+		"required": true,
+	},
+	"date": {
```

Comment:
Combine date and time into one field. We will store the date in ISO format 
eg. `2025-03-06T01:54:14.402Z`